src/sprout/Helpers/Widgets.php

```diff
@@ -116,7 +121,17 @@ public static function render($orientation, $name, array $settings, $pre_html =
 
         $ret = '';
         $ret .= $pre_html;
-        if ($title) $ret .= "<h3 class=\"widget-title\">{$title}</h3>";
+
+        if (!empty($title)) {
+            $heading_html = '<h2 class="widget-title">TITLE</h2>';
+
+            if (!empty(Kohana::config('sprout.widget_title'))) {
+                $heading_html = Kohana::config('sprout.widget_title');
+            }
+
+            $ret .= str_replace('TITLE', $title, $heading_html);
```

Comment:
XSS here. You need `Enc::html($title)` in the replace part

Reply:
Okay so there is a Enc::html on $title above... but that's non-obvious so move it down here.